admin/src/utils/csvHelper.js

```diff
@@ -36,3 +36,20 @@ export const convertToCsv = (data) => {
         : getRow(keys, data);
     return [keys.join(VALUE_DELIMITER), rows].join(ROW_DELIMITER);
 }
+
+export const convertCsvToObject = (data) => {
+    if (!data) return '';
+    const objectRows = data.split('\r\n')
+    const objectKeys = objectRows[0].split(',')
```

Comment:
Split same way as the rows otherwise you'll get fancy results (with delimiter ´;´)